src/rundeck/templates/rundeck-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.volumeclaim.enabled}}
```

Comment:
This must be dropped too in favor of the StatefulsetPattern.

Reply:
Dropped.